DocumentManager.js

```diff
@@ -9,21 +9,52 @@ module.exports = class {
         };
     }
 
+    /**
+   * Recursively appends docIds via pagination
+   */
+    async recurseThroughPagination(cursor, docIds) {
+        let response;
+        try {
+            response = await axios.post(
+                "https://api.dropboxapi.com/2/paper/docs/list/continue",
+                {
+                    cursor: cursor
+                },
+                {
+                    headers: this.baseHeaders
+                }
+            );
+        } catch (e) {
+            console.error(e);
+            return docIds;
+        } finally {
```

Comment:
That makes perfect sense. I had considered making a change like that but it's always tricky when coming onto something because I didn't want to overstep my bounds and start making broad sweeping changes to the existing functions. I am happy to clean that up in each of them!